dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -836,6 +837,7 @@
         #     timer: true
         #     variable: true
         #     variableDocument: true
+        #     form: true
```

Comment:
🔧 I guess this one should be moved up a bit to be in alphabetical order.

Reply:
I'm stupid 😄

Reply:
I disagree 🙂

Reply:
Thanks for spotting this 👍